solidity/contracts/hooks/wormhole/WormholeHook.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+// ============ Internal Imports ============
+import {IPostDispatchHook} from "../../interfaces/hooks/IPostDispatchHook.sol";
+
+// TODO: figure out whether it is possible to import this using Hardhat:
+// https://github.com/wormhole-foundation/wormhole/blob/main/ethereum/contracts/interfaces/IWormhole.sol
+interface IWormhole {
+    function publishMessage(
+        uint32 nonce,
+        bytes memory payload,
+        uint8 consistencyLevel
+    ) external payable returns (uint64 sequence);
+}
+
+contract WormholeHook is IPostDispatchHook {
+    IWormhole public wormhole;
```

Comment:
immutable?

Reply:
Apologies the wormhole hook wasn't meant to be included here it's part of an already existing PR